deploy/kubernetes/base/node.yaml

```diff
@@ -45,7 +45,9 @@ spec:
             - name: efs-state-dir
               mountPath: /var/run/efs
             - name: efs-utils-config
-              mountPath: /etc/amazon/efs
+              mountPath: /var/amazon/efs
+            - name: efs-utils-config-legacy
+              mountPath: /etc/amazon/efs-legacy
```

Comment:
the tests that are failing are all related to the dynamic PR merged just now https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/274.

Could you make the same changes in node.yaml to controller.yaml? shoudl fix it.

Reply:
Thanks I didn't understand what was going on.

Reply:
Sorry, what I suggested isn't right either, the controller yaml makes no reference to any config dir

Let me take a closer look